Week2/Lecture.md

```diff
@@ -59,6 +59,10 @@ $ cat numbers.txt | sort | uniq
 ```
 This will find out all the unique numbers from the file "numbers.txt". Note that, the file must contain numbers on separate lines.
 
+```
+$ cat numbers.txt | grep 3
+```
+This command is the easiest way to find something in the file. It's useful for watching logs, in combination with `tail`
```

Comment:
Could you add a small explanation about `grep` ?

Reply:
Just added one sentence of explanation, should be enought: the teachers already know what is it, ane students know how to `man grep`

Reply:
LGTM.